src/utils/fetchJsonShemas.js

```diff
@@ -0,0 +1,39 @@
+export const getModuleVersion = (modules, moduleName) => {
+  const version = modules.find(module => module.name === moduleName);
+
+  return version ? version.id : undefined;
+};
+
+export const fetchJsonSchema = async (path, module, okapi) => {
+  try {
+    const response = await fetch(`${okapi.url}/_/jsonSchemas?path=${path}`, {
+      headers: {
+        'X-Okapi-Tenant': okapi.tenant,
+        'X-Okapi-Token': okapi.token,
+        'X-Okapi-Module-Id': module,
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error(`Cannot fetch resources from "${path}"`);
+    }
+
+    const data = await response.json();
+
+    return data;
+  } catch (error) {
+    console.error(error); // eslint-disable-line no-console
```

Comment:
Waht is the reason for using `console.error`? You are not processing the error yet?

Reply:
This approach is common for the Data import module